index.html

```diff
@@ -0,0 +1,212 @@
+                </div>           
+                
+                <div class="grid-item">
+                    <div class="profesional"><h4 class="prof-art">Profesional Art Printing Data</h4></div>
+                    <div class="daily"><p>A daily selection of privately personalized reads; no accounts or sign-ups required. Has been the industry's standard.</p></div>
+                    
+                    <ul class="buttons">
+                        <li class="but-html">HTML</li>
+                        <li class="but-boot">Bootstrap</li>
+                        <li class="but-ruby">Ruby</li>
+                    </ul>
+                    
+                    <div class="orange-button1">
+                        <button class="see-project1">See project</button>
+                    </div>
+                </div>    
+                                
+                <div class="grid-item">
+                    <div class="profesional"><h4 class="prof-art">Profesional Art Printing Data</h4></div>
+                    <div class="daily"><p>A daily selection of privately personalized reads; no accounts or sign-ups required. Has been the industry's standard.</p></div>
+                    
+                    <ul class="buttons">
+                        <li class="but-html">HTML</li>
+                        <li class="but-boot">Bootstrap</li>
+                        <li class="but-ruby">Ruby</li>
+                    </ul>
+                    
+                    <div class="orange-button1">
+                        <button class="see-project1">See project</button>
+                    </div>
+                </div>
+
+                <div class="grid-item">
+                    <div class="profesional"><h4 class="prof-art">Profesional Art Printing Datas</h4></div>
+                    <div class="daily"><p>A daily selection of privately personalized reads; no accounts or sign-ups required. Has been the industry's standard.</p></div>
+                    
+                    <ul class="buttons">
+                        <li class="but-html">HTML</li>
+                        <li class="but-boot">Bootstrap</li>
+                        <li class="but-ruby">Ruby</li>
+                    </ul>
+                    
+                    <div class="orange-button1">
+                        <button class="see-project1">See project</button>
+                    </div>
+                </div>            
+            </div>
+        
+        
+    </section>
+    
+    <section id="skillset">
+        <div class="about">
+                <h4>About me</h4>
+                <p>Hello I’m a software developer! I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, don’t hesitate to contact me..</p>
+                    <div class="orange-button2">
+                        <button class="see-project2">Get my resume</button>
```

Comment:
👍 Great job on implementing the button with animation! I really appreciate that you included styles for the active and hover states, which adds interactivity and enhances the user experience. However, there are a few adjustments needed as the current styles don't match the provided design. Here are some suggestions:

- [ ] [Optional] Pay attention to the colors: Make sure the background color, text color, and any border or shadow effects are consistent with the design specifications. The colors play a significant role in maintaining visual harmony and brand consistency.
- [ ] [Optional] Check the size and shape: Verify that the button's dimensions and proportions align with the design. Pay attention to the width, height, and border-radius to ensure they match the intended appearance.

![image](https://github.com/sinruidotec/Portfolio-part-2-Microverse/assets/55840999/a86d36a4-2c47-499f-9d88-58f61f356a94)

By implementing these adjustments, you'll be able to create a button that closely resembles the provided design and maintains a cohesive visual experience. Keep up the great work! 👏🎨